test_resources/serialization_baseline/collections/M-Q4pcV0qkiyJ0kiSWECl_some_collection/cards/OMuZ0wHe2O5Z_59-cLmn4_series_question_a.yaml

```diff
@@ -28,6 +28,6 @@ serdes/meta:
   model: Card
 archived_directly: false
 dashboard_id: null
-metabase_version: v1.54.4-SNAPSHOT (c6780bb)
+metabase_version: v0.55.1-SNAPSHOT (dcb72f1)
```

Comment:
It's fine to scrap the changes to these keys, they don't affect the test (and it's nice to have less noise)